src/examples/avatar/AvatarType.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as UserStrokeIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin-bottom: ${p => p.theme.space.sm};
+  text-align: center;
+`;
+
+const Example = () => {
+  const iconColor = DEFAULT_THEME.palette.grey[600];
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <StyledCol>
+          <Avatar backgroundColor={iconColor} size="large">
+            <UserStrokeIcon role="img" aria-label="User" />
+          </Avatar>
+        </StyledCol>
+        <StyledCol>
+          <Avatar backgroundColor={iconColor} size="large">
+            <img
+              alt="avatar"
```

Comment:
`alt="image avatar"`

Reply:
ESLint is saying the word "image" would be redundant.

<img width="891" alt="Screen Shot 2020-05-12 at 11 06 45 AM" src="https://user-images.githubusercontent.com/1811365/81855733-9468c480-9514-11ea-8d76-13f8c9f6a88a.png">

Reply:
Maybe something like `Example avatar`? The `img` tag will read as "image" already

Reply:
It has been updated to "example avatar". Let me know if there's another suggestion 👍.